### DISCUSSION TITLE
Is it possible to have a check for authors who are part of the particular organization

### DISCUSSION THREAD

**Discussion by yadavan88** [Mergifyio/mergify#5103] [Q&A] [answered]
Hi,
I tried searching, but could not find it. I read that it is possible to use @teamname in the author field for the rule. 
Similarly, is it possible to check if the PR author is a part of an organization. 

Just to give the scenario:
I want to auto-merge the PRs that are for README.md markdown files under any sub-directories that are created by any persons who are part of the organization. That person might not have write access to the particular repo.  

Thanks in advance

**Comment by jd** [comment, marked as answer]
That's not supported for now, but that is something we could probably add at some point.
In the meantime, you could probably leverage a GitHub team that include every member of the org.
